File: XXTExplorer/SKHelperConfig.m
//
// Created by Zheng on 18/08/2017.
// Copyright (c) 2017 Zheng. All rights reserved.
//

#import "SKHelperConfig.h"


@implementation SKHelperConfig {

}

- (instancetype)init {
    self = [super init];
    if (self)
    {
        _bundle = [NSBundle mainBundle];
        _font = [UIFont systemFontOfSize:14.f];
        _color = UIColor.blackColor;
        _themeIdentifier = @"";
        _languageIdentifier = @"";
    }
    return self;
}

@end

File: XXTExplorer/SKHelperConfig.swift
//
//  SKHelperConfig.swift
//  XXTExplorer
//
//  Created by Zheng Wu on 11/08/2017.
//  Copyright © 2017 Zheng. All rights reserved.
//

import Foundation
import UIKit

@objc(SKHelperConfig)
open class SKHelperConfig : NSObject {
    public var bundle: Bundle = Bundle.main
    public var font: UIFont = UIFont.systemFont(ofSize: 14)
    public var color: UIColor = UIColor.black
    public var path: String = ""
    public var themeIdentifier: String = ""
    public var languageIdentifier: String = ""
}
